### PR TITLE
Add statistics card

### LINKED ISSUE #24: Create red flags in the last 24h component
**Prototype**
https://www.figma.com/file/rdg8gZtSWNYaVtupMtdhST/CoVid---Stam-Acasa?node-id=144%3A10

**Design**
Screenshot 2020-03-26 at 23 49 49

### LINKED ISSUE #25: Number of users logged in
**Prototype**
https://www.figma.com/file/rdg8gZtSWNYaVtupMtdhST/CoVid---Stam-Acasa?node-id=144%3A10

**Design**
Screenshot 2020-03-26 at 23 51 03

### LINKED ISSUE #26: Number of filled in forms component
**Prototype**
https://www.figma.com/file/rdg8gZtSWNYaVtupMtdhST/CoVid---Stam-Acasa?node-id=144%3A10

**Design**
Screenshot 2020-03-26 at 23 51 16

### PR DESCRIPTION
**Pull request by Patricia008**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes #24 
Closes #25 
Closes #26 

Please mention the main changes this PR brings.

Adds the card component to show dashboard data

### How has it been tested?

no tests, I've created a new route to display it: http://localhost:3000/cards-playground

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

File: frontend/src/components/StatisticsCard/CircularProgressBar/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        viewBox="0 0 44 43"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          opacity="0.518059"
+          d="M34.5928 42.5461C34.5928 42.5461 32.8283 42.4435 36.7358 40.6932C40.6433 38.9429 47.8156 23.7465 40.0484 19.9222C32.2811 16.0978 39.5443 8.42575 36.8257 4.08655C34.1071 -0.252643 31.1689 -0.628618 24.7196 3.65805C18.2703 7.94472 5.82833 -3.50456 1.47175 5.58396C-2.88484 14.6725 4.51531 3.63802 4.51531 3.63802L22.6606 9.34632L34.1045 19.3672L34.5928 42.5461Z"
+          fill="#F2994A"
+        />
+      </svg>
```
Comment (surdu) [inline frontend/src/components/StatisticsCard/CircularProgressBar/index.js]: I'm guessing that these abstract shapes where intended by @overeha to represent another dimension of the data. I propose to comment them out for now, as they might add confusion

**Review by surdu — CHANGES_REQUESTED** [review]

File: frontend/src/components/StatisticsCard/CircularProgressBar/index.js (hunk 1 above)
Comment (Patricia008) [reply frontend/src/components/StatisticsCard/CircularProgressBar/index.js]: okay, removing them

File: frontend/src/routes.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import PostLogoutRoute from "./components/OIDC/PostLogoutRoute";
 import SilentRefreshRoute from "./components/OIDC/SilentRefreshRoute";
 import Login from "./components/Login";
 import CardsPlayground from "./components/CardsPlayground";
+import AddMember from "./components/AddMember";
```
Comment (surdu) [inline frontend/src/routes.js:7]: Hmmm... Why is this in this PR?

Comment (Patricia008) [reply frontend/src/routes.js:7]: no idea, really...locally I don't have it as a difference, it seems like it was commited 5 days ago by someone else

Comment (surdu) [reply frontend/src/routes.js:7]: Nevermind ... It shows only on "What's changed" view ... My bad

**Review by surdu — APPROVED** [review]

File: frontend/package-lock.json [hunk 3]
```diff
@@ -1290,6 +1290,100 @@
         "glob-to-regexp": "^0.3.0"
       }
     },
+    "@nivo/axes": {
```
Comment (surdu) [inline frontend/package-lock.json:1293]: There seems to be a syntax error in the `package-lock.json` file. You might want to delete it and run `npm install` again to re-create it

**Review by surdu — CHANGES_REQUESTED** [review]

**Review by surdu — APPROVED** [review]

**Comment by surdu** [comment]
@Patricia008 Could you please re-build the package-lock file one last time? Sorry for the inconvenience 😞

**Comment by surdu** [comment]
Unknowingly of a similar effort, this was addressed and solved in #93. Closing this for now. Thank you @Patricia008 for you awesome work! 💪

**Pull request closed without merging**